### ISSUE TITLE
Question - Is it possible to ignore fields on dump?

### ISSUE THREAD

**Issue by sanketatadvisor** [marshmallow-code/marshmallow#2185]
Following up on this discussion - https://github.com/marshmallow-code/flask-marshmallow/issues/50

I would like to not exclude, but ignore the fields on dump. Is that possible? 

```
class Account(BaseSchema):
  id= fields.Int(required=False, allow_none=True, data_key='accountId')
   number= fields.Nested(DataMask, required=False, allow_none=False, data_key='accountNumber')
```

On

`Account().dump()`, it throws an error because number is a DataMask object that it cant serialize. I want the output to be -

```
{"accountId": 123456789,
    "accountNumber": {
      "isMasked": true,
      "canChange": false,
      "encryptedValue": "23sajds92!@",
      "maskedValue": "***45*",
      "rawValue": "12345",
      "category": {
        "name": "string"
      }
    }
}
```

I want the field to be ignored when dumping, not excluded. Exclusion works OK, but doesn't print the value.

Thanks in advance.

**Comment by lafrech** [comment]
I don't understand what you're asking for, which field you want to ignore/exclude on dump, why number couldn't be serialized but still is serialized at least partially in the output.

**Comment by sanketatadvisor** [comment]
The DataMask class is

```
class DataMask:
  isMasked = fields.Bool()
  canChange = fields.Bool()
 encryptedValue = fields.Str()
maskedValue = fields.Str()
rawValue = fields.Str()
category = fields.Nested(Category)

class Categor:
 name = fields.Str()
```

When I try to serialize, my output for dump is as such -

```
`'{"accountId": 123456789,
    "accountNumber": {
      "category": {
        "name": "AccountNumber"
      }
    }
}`
```

The other fields(isMasked, etc)  are missing on serialization. I am not sure why or if I have to add any attributes here. I was thinking ignoring might help, if its possible, so as to get the entire number field as expected.

TLDR: The serialization is ignoring some fields in the number field. Not sure why that is the case.

**Comment by lafrech** [comment]
Please provide a reproducible example.

**Comment by sanketatadvisor** [comment]
I was just doing it incorrectly. Apologies.
